Development installation instructions for Ubuntu are missing tkinter

## Description
OS-specific installation instruction for Ubuntu is missing tkinter module. 
Although according to the docs tkinter is part of the standard library it does not seem to be always shipped with the core python installation. Missing tkinter module is causing that unit tests are failing. 

## How to reproduce
* follow the installation instructions Install.rst to install scikit-image on Ubuntu with vanilla Python3
* from `scikit-image` directory run tests, e.g. `pytest skimage/filtering`
* `skimage/filters/tests/test_thresholding.py` test fails with
```
>   import tkinter as Tk
E   ModuleNotFoundError: No module named 'tkinter'
```

### Solution
`sudo apt-get install python3-tk`

--- a/INSTALL.rst
+++ b/INSTALL.rst
@@ -90,7 +90,7 @@ Install all the required dependencies::
 
 Install suitable compilers::
 
-  sudo apt-get install build-essential cython
+  sudo apt-get install build-essential cython3
 
 Complete the general development installation instructions above.
 

--- a/INSTALL.rst
+++ b/INSTALL.rst
@@ -86,7 +86,7 @@ b. Debian and Ubuntu
 ````````````````````
 Install all the required dependencies::
 
-  sudo apt-get install python-matplotlib python-numpy python-pil python-scipy
+  sudo apt-get install python-matplotlib python-numpy python-pil python-scipy python3-tk
 
 Install suitable compilers::
 

--- a/INSTALL.rst
+++ b/INSTALL.rst
@@ -86,7 +86,7 @@ b. Debian and Ubuntu
 ````````````````````
 Install all the required dependencies::
 
-  sudo apt-get install python-matplotlib python-numpy python-pil python-scipy python3-tk
+  sudo apt-get install python3-matplotlib python3-numpy python3-pil python3-scipy python3-tk
 
 Install suitable compilers::
 

